src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -57,7 +57,9 @@ export class CdpTarget {
 
   #deviceAccessEnabled = false;
   #cacheDisableState = false;
-  #networkDomainEnabled = false;
+  // Even though the CDP `Network` event should be always enabled, some additional domains
+  // can be or be not required.
+  #networkBidiDomainEnabled = false;
```

Comment:
do we need to rename it? I think we are still toggling the CDP's network domain and not the bidi network domain.

Reply:
I meant "Enable all required for BiDi Network Module"

Reply:
Actually we don't need this field at all

Reply:
reverted